Sources/DataPipeline/CIODataPipelineConfig.swift

```diff
@@ -5,8 +5,12 @@ import Segment
 extension Configuration {
     // TODO: Module Level configuration is going to be added here which will replace this config
     static var defaultConfiguration: Configuration {
-        let siteId = CustomerIO.shared.config?.siteId ?? ""
-        let apiKey = CustomerIO.shared.config?.apiKey ?? ""
+        configure(diGraph: CustomerIO.shared.diGraph)
+    }
+
+    static func configure(diGraph: DIGraph?) -> Configuration {
+        let siteId = diGraph?.sdkConfig.siteId ?? ""
+        let apiKey = diGraph?.sdkConfig.apiKey ?? ""
         let key = siteId + apiKey
         return Configuration(writeKey: key)
```

Comment:
I'm wondering if there are plans to add support for `writeKey` in future tickets or pull requests. It seems like this implementation is designed for backward compatibility, but what happens if a user only provides `writeKey` and not `siteId` or `APIKey`?

Reply:
This is going to be replaced with `write key` the comment on the top,  

```
// TODO: Module Level configuration is going to be added here which will replace this config
```

`Module Level configuration` this module config would contain write key

Reply:
Perfect, Thanks !